README.md

```diff
@@ -25,6 +25,7 @@ If you have any other questions not listed here, please raise a PR to have it ad
 - Q: How can I `add` only parts of a file I've changed?
 
 - Q: How can I __revert__ a commit?
+It looks like ```git revert HEAD``` will undo the last commit but then commit the last changes as a new commit. The previous commits (both before and after the last commit) are still in the history. I think!!!
```

Comment:
correct! How would you revert a specific commit? Hint: `HEAD` is another word for `the most reset commit`, and the answer contains a hash ;)